addons/sale/models/sale_order.py

```diff
@@ -212,7 +212,11 @@ def _rec_names_search(self):
     amount_tax = fields.Monetary(string="Taxes", store=True, compute='_compute_amounts')
     amount_total = fields.Monetary(string="Total", store=True, compute='_compute_amounts', tracking=4)
     amount_to_invoice = fields.Monetary(string="Amount to invoice", store=True, compute='_compute_amount_to_invoice')
-    amount_invoiced = fields.Monetary(string="Already invoiced", compute='_compute_amount_invoiced')
+    amount_invoiced = fields.Monetary(
+        string="Already invoiced",
+        compute='_compute_amount_invoiced',
+        compute_sudo=True,  # allow overrides to check `invoice_ids`
```

Comment:
Mhm, I don't like having to sudo something for overrides we shouldn't be aware of.
Why don't you add a sudo for the field access in the override instead ? :thinking:

Reply:
They are only because they're stored, whereas this one isn't, but I agree the reasoning is valid.
I'd make the comment more generic though, to explain that you want to be more generic and behave like the other `_compute_amount*`, instead of speaking of 'invoice_ids' specifically.